### DISCUSSION TITLE
Does aioble now work on the Pico?

### DISCUSSION THREAD

**Discussion by Guustaaf** [micropython/micropython#11791] [RP2040 / Pico] [answered]
And is that the recommended library for that?

Thanks.

**Comment by mattytrentini** [comment, marked as answer]
Yes, and yes. 😄

**Reply by Guustaaf** [reply]
Another feather in the cap of MicroPython!

**Comment by Guustaaf** [comment]
I am using VScode on Windows with Pico W Go

Should this normally work?

>>> import mip
>>> mip.install("aioble")
Installing aioble (latest) from https://micropython.org/pi/v2 to /lib
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "mip/__init__.py", line 1, in install
  File "mip/__init__.py", line 1, in _install_package
  File "mip/__init__.py", line 1, in _install_json
  File "urequests.py", line 180, in get
  File "urequests.py", line 76, in request
OSError: -6

**Reply by jimmo** [reply]
For anyone else reading this see https://github.com/orgs/micropython/discussions/11792#discussioncomment-6192763

**Comment by brianreinhold** [comment]
Pairing is not supported for the btstack which is used by the PICO W, so that will take quite a bit of work to add that support.
